source/concepts/aggregate-transaction.rst

```diff
@@ -6,23 +6,53 @@ Aggregate Transaction
 
 Aggregated Transactions merge multiple transactions into one, allowing **trustless swaps**, and other advanced logic. NEM does this by generating a one-time disposable smart contract. When all involved :doc:`accounts<../concepts/account>` have cosigned the aggregate transaction, all the inner transactions are executed at the same time.
 
-********
-Examples
-********
+.. _aggregate-complete:
+
+******************
+Aggregate complete
+******************
+
+An aggregate transaction is  **complete** when all cosigners (for multisignature account) have signed it.
```

Comment:
Please edit ``when all cosigners (for multisignature account)`` for ``when all the required participants``.